tests/integration/test.bulk_docs.js

```diff
@@ -162,6 +162,23 @@ adapters.forEach(function (adapter) {
       });
     });
 
+    if (adapter === 'local') {
```

Comment:
Is `1-` a valid rev in CouchDB? We very much try to avoid needing these type of switches and having consistent behaviour between couch and pouch

Reply:
> Is `1-` a valid rev in CouchDB?

Yes, it is. This is actually how I discovered the PouchDB bug--I was trying to sync a `1-` rev from CouchDB to PouchDB, when Pouch failed.

Reply:
Maybe that switch isn't necessary. I now can't remember why I put it there. Maybe related to my local test environment. Let me investigate.

Reply:
Well, without that switch, the test passes, too, so I've removed it and re-pushed that commit.

Reply:
Awesome, I think you need to push the commit here though, it still exists in this PR